com.unity.render-pipelines.core/Editor/CoreEditorUtils.cs

```diff
@@ -88,6 +89,36 @@ public static Texture2D CreateColoredTexture2D(Color color, string textureName)
             return tex2;
         }
 
+        /// <summary>Loads a <see cref="Texture2D"/> from within the path console.*</summary>
+        /// <param name="isError">If the texture to load is the error one, otherwise it will be warning texture</param>
+        /// <returns>a <see cref="Texture2D"/></returns>
+        public static Texture2D LoadConsoleIcon(bool isError)
+        {
+            string pathToIcon = "icons/";
+
+            // Handle different skin
+            if (EditorGUIUtility.isProSkin)
+                pathToIcon += "d_";
+
+            // Handle different icon
+            if (isError)
+                pathToIcon += "console.erroricon";
+            else
+                pathToIcon += "console.warnicon";
+
+            // Handle different resolution
+            if (EditorGUIUtility.pixelsPerPoint > 1.0f)
+            {
+                pathToIcon += "@2x";
+            }
+
+            pathToIcon += ".png";
+
+            Texture2D icon = EditorGUIUtility.Load(pathToIcon) as Texture2D;
+
+            return icon;
+        }
+
```

Comment:
Maybe not have a class just for 2 textures

Reply:
What happen if the icon doesnt exist? or what if it doesnt have d_ version or a @2x version?

Reply:
a console error will be shown, and the icon will be missing.